tests/test_cli_client.cpp

```diff
@@ -1392,6 +1403,31 @@ TEST_F(Client, set_handles_persistent_settings_errors)
     EXPECT_THAT(send_command({"set", keyval_arg(key, val)}), Eq(mp::ReturnCode::CommandFail));
 }
 
+TEST_F(Client, get_returns_acceptable_autostart_value_by_default)
+{
+    EXPECT_THAT(get_setting(mp::autostart_key), AnyOf("true", "false"));
```

Comment:
Are you testing the acceptance of "on", "off" etc. somewhere?

Reply:
No, so far there are no tests checking acceptance/rejection of settings values unfortunately :/  This is because, to do that, I would need another wrapping layer around the `QSettings` backend, in order to mock it. That would be a considerable investment that I have been avoiding so far.

Reply:
Ack.